runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -210,6 +216,18 @@ task buildAndPushDockerContainer() {
   }
 }
 
+// Clean up built images
+task cleanUpDockerImages() {
```

Comment:
Any reason for relocating this task from L448?

Reply:
Sort of :-)

I had a syntax error and the error was "cannot find task 'cleanUpdockerImages'". So I moved this task to an earlier line and the error was easier to understand.

I was going to revert it, but I noticed it finalized the task `buildAndPushDockerContainer` which is right above it. So I thought it was kind of nice to have them close to each other. That is why I left it here.

Reply:
Make sense. Thanks.